MediaGalleryUi/view/adminhtml/web/js/image/image-edit.js

```diff
@@ -100,6 +100,25 @@ define([
         addMediaGridMessage: function (code, message) {
             this.mediaGridMessages().add(code, message);
             this.mediaGridMessages().scheduleCleanup();
+        },
+
+        /**
+         * Handle Enter key event to save image details
+         *
+         * @param {Object} data
+         * @param {jQuery.Event} event
+         */
+        handleEnterKey: function (data, event) {
```

Comment:
I think we can use`Magento_Ui/js/lib/key-codes` instead of the 13, it will make it easier to understand. Do we need to return anything on this function? I think void would be cleaner and easier to understand.

Something like this:

```
        handleEnterKey: function (data, event) {
            var modalElement = $(this.modalSelector),
                key = keyCodes[event.keyCode];

            if (key === 'enterKey') {
                event.preventDefault();
                modalElement.find('.page-action-buttons button.save').click();
            }
        }
```

What do you think?

Reply:
Hi @gabrieldagama, thank you for reviewing my pull request. I just added the return for all keycodes other than the enter key since if there's no return, the user can no longer input anything in the input fields but I'll try your suggestion with the void. I'll just update my pull request. Thank you again.

Reply:
Hi @gabrieldagama, I tried to work with a void but I had a problem with the input field where I can't type/input anything unless I add a return true. I updated my pull request with the requested changes. Thank you.